### DISCUSSION TITLE
Resource limits

### DISCUSSION THREAD

**Discussion by ahmed-abdelazim** [acorn-io/runtime#555] [Ideas]
Hello
Anyway to set resource limits / memory lock in Acornfile?
also anyway to set endpoint host / domain in Acornfile?

**Comment by ibuildthecloud** [comment]
Regarding resources limits, we haven't added that yet but it's definitely coming.

**Reply by ibuildthecloud** [reply]
What exactly do you mean by endpoint/hostname? Is that the same as https://github.com/acorn-io/acorn/issues/512

**Reply by ahmed-abdelazim** [reply]
Yes. https://github.com/acorn-io/acorn/issues/512
Thank you!

**Comment by ibuildthecloud** [comment]
Can you describe a bit more about memory lock and what are the issues you are encountering?

**Reply by ibuildthecloud** [reply]
The only thing I could find in reference to this is elasticsearch wanting to lock pages in memory to disable swapping.  Most k8s setups do not have swap, so swapping out isn't really an issue.  If you are running elasticsearch or similar program that uses mlockall, can you disable that as it's not really needed on k8s, I believe.  Correct me if I'm wrong.

**Reply by ahmed-abdelazim** [reply]
I tried to run an Elasticsearch cluster but cannot run multiple nodes. I think memlock is mandatory in this case.
